Core/GDCore/Extensions/Builtin/MathematicalToolsExtension.cpp

```diff
@@ -492,6 +492,16 @@ BuiltinExtensionsImplementer::ImplementsMathematicalToolsExtension(
                      "",
                      "res/mathfunction.png")
       .SetHelpPath("/all-features/expressions");
+
+  extension
+      .AddExpression("lerp_angle",
```

Comment:
Can you use camelCase for the expression and function names? There's currently a lack of consistency across all expressions, but I think that camelCase is the most used.